src/utils/mongodb.ts

```diff
@@ -0,0 +1,50 @@
+// Source: Official Next.js MongoDB example
+// https://github.com/vercel/next.js/blob/master/examples/with-mongodb/util/mongodb.js
+
+import { MongoClient } from 'mongodb'
+
+// Required to use global._mongoClientPromise
+declare global {
+  /* eslint-disable-next-line */
+  namespace NodeJS {
+    interface Global {
+      _mongoClientPromise: Promise<typeof MongoClient>
+    }
+  }
+}
+
+const uri = process.env.MONGO_URL || ''
+const host = process.env.MONGO_HOST || ''
+const user = process.env.MONGO_USER || ''
+const password = process.env.MONGO_PASSWORD || ''
+
+// Connection string for DocumentDB
+let connectionString = `mongodb://${user}:${password}@${host}/?tls=true&tlsCAFile=rds-combined-ca-bundle.pem&replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false`
```

Comment:
nit: I think this could be made a tad bit cleaner with
```
const connectionString = process.env.MONGO_URL || `mongodb://${user}:${password}@${host}/?tls=true&tlsCAFile=rds-combined-ca-bundle.pem&replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false`
```

then `MongoClient` can always be called with the same value regardless of the NODE_ENV (which isn't really relevant)

```
client = new MongoClient(connectionString)
```